doc/sr_cpp_walkthrough.rst

```diff
@@ -3,10 +3,9 @@ C++
 ***
 
 
-In this section, examples are presented using the SmartRedis C++
-API to interact with the RedisAI tensor, model, and script
-data types.  Additionally, an example of utilizing the
-SmartRedis ``DataSet`` API is also provided.
+This section presents examples that use the SmartRedis C++ API to 
+interact with the RedisAI tensor, model, and script data types. 
+Additionally, this section demonstrates an example of utilizing the SmartRedis ``DataSet`` API.
```

Comment:
Could you make sure the line lengths follow black convention? (88 characters per line)